README.md

```diff
@@ -1,324 +1,47 @@
-# Strimzi : Kafka as a Service
+# Strimzi: Kafka as a Service
 
 [![Build Status](https://travis-ci.org/strimzi/strimzi.svg?branch=master)](https://travis-ci.org/strimzi/strimzi)
 [![License](https://img.shields.io/badge/license-Apache--2.0-blue.svg)](http://www.apache.org/licenses/LICENSE-2.0)
 [![Twitter Follow](https://img.shields.io/twitter/follow/strimziio.svg?style=social&label=Follow&style=for-the-badge)](https://twitter.com/strimziio)
 
 Strimzi provides a way to run an [Apache Kafka][kafka] cluster on 
 [Kubernetes][k8s] or [OpenShift][os] in various deployment configurations.
+See our [website][strimzi] for more details about the project.
 
-<!-- TOC depthFrom:2 -->
+## Documentation
 
-- [Deploying a Kubernetes/OpenShift cluster](#deploying-a-kubernetesopenshift-cluster)
-    - [Kubernetes](#kubernetes)
-    - [OpenShift](#openshift)
-- [Cluster Controller](#cluster-controller)
-    - [Deploying to OpenShift](#deploying-to-openshift)
-    - [Deploying to Kubernetes](#deploying-to-kubernetes)
-- [Kafka broker](#kafka-broker)
-    - [Deploying to OpenShift](#deploying-to-openshift-1)
-    - [Deploying to Kubernetes](#deploying-to-kubernetes-1)
-- [Kafka Connect](#kafka-connect)
-    - [Deploying to OpenShift](#deploying-to-openshift-2)
-    - [Deploying to Kubernetes](#deploying-to-kubernetes-2)
-    - [Using Kafka Connect with additional plugins](#using-kafka-connect-with-additional-plugins)
-        - [Create a new image based on `strimzi/kafka-connect`](#create-a-new-image-based-on-strimzikafka-connect)
-        - [Using OpenShift Build and S2I image](#using-openshift-build-and-s2i-image)
-- [Metrics](#metrics)
+Documentation to the current _master_ branch as well as all releases can be found on our [website][strimzi].
 
-<!-- /TOC -->
+## Getting help
 
-## Deploying a Kubernetes/OpenShift cluster
+If you encounter any issues while using Strimzi, you can get help using:
 
-You will need a Kubernetes or OpenShift cluster to deploy Strimzi.
+- [Strimzi mailing list](https://www.redhat.com/mailman/listinfo/strimzi)
+- [#strimzi IRC channel](https://webchat.freenode.net/?randomnick=1&channels=strimzi&uio=d4) on Freenode
 
-### Kubernetes
+## Contributing
 
-In order to interact with a Kubernetes cluster, be sure to have [the kubectl tool][kubectl] installed.
-If you don't have a Kubernetes cluster up and running, the easiest way to deploy one, for development purposes,
-is to use the [Minikube][minikube] project which provides a single-node cluster in a VM. In order to do that you
-can just follow the installation guide which describes all the prerequisites and how to get the Minikube binaries.
-Finally, the cluster can be deployed by running
+You can contribute by:
+- Raising issues
```

Comment:
Raising any issues you find using strimzi
Fixing issues by opening Pull Requests